app/views/katello/api/v2/content_views/_content_view.json.rabl

```diff
@@ -23,7 +23,7 @@ child :environments => :environments do
 end
 
 child :repositories => :repositories do
-  extends 'katello/api/v2/repositories/show'
+  attributes :id, :name, :label
```

Comment:
it looks like hammer, may also be using 'content_type':  

https://github.com/Katello/hammer-cli-katello/blob/master/lib/hammer_cli_katello/content_view.rb#L79